.github/workflows/test-ui.yml

```diff
@@ -23,6 +23,32 @@ jobs:
         uses: ./.github/actions/setup-vuu-ui
       - run: cd ./vuu-ui && npm run test:vite
 
+  playwright:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+      - name: Install Node & dependencies
+        uses: ./.github/actions/setup-vuu-ui
+      - name: Install dependencies
+        run: npm ci
+      - name: Run Vuu Showcase
+        run: npm run showcase:build && npm run showcase:preview
+      - name: Install Playwright Browsers
+        run: npx playwright install --with deps chromium
```

Comment:
we shouldn't need to use npx, we have playwright in our devDependencies, so it should get installed

npx is only for scenarios where the package you need might not be installed